apps/docs/docs/dev/12-jayvee-testing.md

```diff
@@ -0,0 +1,247 @@
+---
+title: Writing tests for Jayvee
+---
+
+In order to ensure that Jayvee functions correctly a number of tests have been written already.  
+Additionally a few testing utils have been implemented to help writing tests and especially to avoid having to create Langium Typescript objects by hand.
+It has to be mentioned that, like Jayvee itself, the tests are split into grammar and execution tests.
```

Comment:
Rather make a list of different types of tests?

Reply:
What exactly do you mean with that?

Reply:
I mean that the paragraph is quite lengthy to read. I'd prefer a bullet list, sth like this:
```
We implemented the following components for regressionion testing:
- Testing utils: utils to create Langium TypeScript objects by hand
- Grammar tests: test the grammar parsing and validation
- Execution tests: test the execution of blocks
```
And each bullet point could link to the subsection headings below (like a table of contents).